packages/cxl-ui/scss/cxl-featured-course-card.scss

```diff
@@ -3,32 +3,45 @@
 
 :host {
   position: relative;
-  max-width: unset;
+  display: flex;
+  align-items: center;
+  width: auto;
   height: auto;
+  min-height: 400px;
+  max-width: unset;
+  margin: 0 calc(var(--lumo-space-m) * -1);
   padding: 0;
   border: 0;
   box-shadow: none;
+  border-radius: 0;
+  background-color: var(--cxl-color-black);
+
+  @media #{mq.$medium} {
+    width: 100%;
+    margin: 0;
+    padding-right: 16px;
+    padding-left: 16px;
+  }
 
   .container {
-    position: relative;
     display: block;
-    height: 100%;
-    padding: 0 var(--lumo-space-m);
-    background-color: var(--cxl-color-light-gray);
+    margin: 0 auto;
+    padding: 0;
+    color: var(--lumo-tint);
 
     @media #{mq.$medium} {
-      display: grid;
-      grid-template-columns: repeat(12, 1fr);
-      grid-template-rows: min-content auto;
-      gap: 0;
-      padding: var(--lumo-space-l) 0 0 var(--lumo-space-l);
+      padding-right: 400px;
     }
 
     > .attributes {
       display: none;
       padding: 0;
     }
 
+    .name {
+      font-size: 40px;
```

Comment:
I assume we don't have font size var close enough to use here. @freudFlintstone based on your styles sync, are you able to recommend something here?